vendor/k8s.io/kubernetes/staging/src/k8s.io/client-go/tools/record/events_cache.go

```diff
@@ -416,6 +416,24 @@ type EventCorrelateResult struct {
 	Skip bool
 }
 
+type EventCorrelatorOptions struct {
+	cacheSize                  int
```

Comment:
Export these so they can be set outside of this package.  Doc all the fields and the struct itself.